src/plugins/content/accessibility/keyNavigation/strategies/navigatorNavigation.js

```diff
@@ -51,179 +105,187 @@ export default {
         const config = this.getConfig();
         const $panel = this.getTestRunner().getAreaBroker().getPanelArea();
         const $navigator = $panel.find('.qti-navigator');
-        let filtersNavigator;
-        let itemsNavigator;
-
-        //the tag to identify if the item listing has been browsed, to only "smart jump" to active item only on the first visit
-        let itemListingVisited = false;
-        //the position of the filter in memory, to only "smart jump" to active item only on the first visit
-        let currentFilter;
+        const isFizzyLayout = $navigator.hasClass('qti-navigator-fizzy');
 
         this.managedNavigators = [];
         this.keyNavigators = [];
 
-        let testStatusNavigation;
         if ($navigator.length && !$navigator.hasClass('disabled')) {
-            const $filters = $navigator.find(selectors.filters);
-            const navigableFilters = navigableDomElement.createFromDoms($filters);
-            const $testStatusHeader = $navigator.find('.qti-navigator-info.collapsible > .qti-navigator-label');
-            const navigableTestStatus = navigableDomElement.createFromDoms($testStatusHeader);
-
-            $testStatusHeader.addClass('key-navigation-actionable');
-
-            if (navigableTestStatus.length) {
-                testStatusNavigation = keyNavigator({
-                    keepState: config.keepState,
-                    id: 'navigator-test-status',
-                    propagateTab: false,
-                    elements: navigableTestStatus,
-                    group: $testStatusHeader,
-                });
-
-                setupItemsNavigator(testStatusNavigation, {
-                    keyNextItem: config.keyNextTab || config.keyNextItem,
-                    keyPrevItem: config.keyPrevTab || config.keyPrevItem
-                });
-
-                this.keyNavigators.push(testStatusNavigation);
-                this.managedNavigators.push(testStatusNavigation);
+            if (isFizzyLayout) {
+                initFizzy(config, $panel, $navigator, this.keyNavigators, this.managedNavigators);
             }
+            else {
+                let filtersNavigator;
+                let itemsNavigator;
```

Comment:
`const` won't work, because they are checked late in both `if (navigableFilters.length) {` and `if (navigableTrees.length) {` branches, so they should be declared in outer scope. Maybe it can be refactored further, but this code already existed before, so I'm afraid to change it too much :) 
Only moved declarations below ` if (navigableTestStatus.length) {`.